### ISSUE TITLE
Low accuracy rate

### ISSUE THREAD

**Issue by slighting666** [okankop/Efficient-3DCNNs#13]
I use the pre training model to fine tune the ucf101, i use the video_accuracy.py. but the accuracy of the video is only 60.2%. The accuracy of your paper is 76%. What did I miss? Here is my opts. JSON. Thanks!

{"root_path": "/3DCNN/3DCNN/",
"video_path": "/3DCNN/3DCNN/data/jpg/", "annotation_path": "/3DCNN/3DCNN/data/UCF101TrainTestSplits-RecognitionTask/ucfTrainTestlist/ucf101_01.json"
 "result_path": "/3DCNN/3DCNN/results",
"store_name": "ucf101_shufflenet_1x_RGB_16", "modality": "RGB", 
"dataset": "kinetics", "n_classes": 600, "n_finetune_classes": 101, "sample_size": 112, "sample_duration": 16, "downsample": 1, "initial_scale": 1.0, "n_scales": 5, "scale_step": 0.84089641525, "train_crop": "random", "learning_rate": 0.1, "lr_steps": [40, 55, 65, 70, 200, 250], "momentum": 0.9, "dampening": 0.9, "weight_decay": 0.001, "mean_dataset": "activitynet", "no_mean_norm": false, "std_norm": false, "nesterov": false, "optimizer": "sgd", "lr_patience": 10, "batch_size": 128, "n_epochs": 250, "begin_epoch": 1, "n_val_samples": 1, "resume_path": "", "pretrain_path": "kinetics_shufflenet_1.0x_G3_RGB_16_best.pth", "ft_portion": "last_layer", "no_train": false, "no_val": false, "test": false, "test_subset": "val", "scale_in_test": 1.0, "crop_position_in_test": "c", "no_softmax_in_test": false, "no_cuda": false, "n_threads": 16, "checkpoint": 1, "no_hflip": false, "norm_value": 1, "model": "shufflenet", "version": 1.1, "model_depth": 50, "resnet_shortcut": "B", "wide_resnet_k": 2, "resnext_cardinality": 32, "groups": 3, "width_mult": 1, "manual_seed": 1, "scales": [1.0, 0.84089641525, 0.7071067811803005, 0.5946035574934808, 0.4999999999911653], "arch": "shufflenet", "mean": [114.7748, 107.7354, 99.475], "std": [38.7568578, 37.88248729, 40.02898126]}

**Comment by okankop** [comment]
I have retrained with your configuration and clip accuracy for shufflenetv1_1.0x reaches to 70% and its video accuracy is 76%. Can you train again with "lr_steps": [20, 40, 55, 65]"?

**Comment by slighting666** [comment]
thanks！
